components/sysutils/virtualbox/virtualbox.p5m

```diff
@@ -32,3 +137,108 @@ file path=platform/i86pc/kernel/drv/vboxflt.conf
+file path=usr/share/icons/hicolor/32x32/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/32x32/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/32x32/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/32x32/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/32x32/mimetypes/virtualbox-vmdk.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-hdd.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-ova.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-ovf.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vmdk.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-hdd.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-ova.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-ovf.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vmdk.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-hdd.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-ova.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-ovf.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vmdk.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-hdd.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-ova.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-ovf.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vmdk.png
+file path=usr/share/mime-info/virtualbox.keys
+file path=usr/share/mime-info/virtualbox.mime
+file path=usr/share/mime/packages/virtualbox.xml
+file path=usr/share/pixmaps/VBox.png
+file path=usr/share/pixmaps/virtualbox.png
+file path=var/svc/manifest/application/virtualbox/virtualbox-autostart.xml
```

Comment:
Should we place it in /lib/svc/manifest ?

Reply:
OK